rotkehlchen/chain/ethereum/graph.py

```diff
@@ -82,7 +89,7 @@ def _maybe_create_client(self) -> Client:
                     },
                 )
                 self.warning_given = True
-
+        elif api_key is None:  # don't bother the user and fail silently. We avoid exhausting the default key  # noqa: E501
```

Comment:
is this ApiKeyNotConfigured exception handled in a way that it just skips the query? I guess yes but need to confirm

Reply:
Yes, in the ens decoder and in balancer, for example here https://github.com/rotki/rotki/blob/3856b9b9f5a7ce740511618cc2e5b31f93ce8333/rotkehlchen/chain/ethereum/modules/balancer/balancer.py#L119 but wherever it is called